tests/network-ovn

```diff
@@ -2096,6 +2096,51 @@ ovn_leases_tests() {
         lxc network delete lxdbr0
 }
 
+ovn_vlan_uplink_tests() {
```

Comment:
This test will need to be called (at the bottom of the script) if the right API extension is detected. See `hasNeededAPIExtension` in `bin/helpers`.

Reply:
Added